src/v1/components/data-display/UserProfile/UserProfile.tsx

```diff
@@ -25,36 +29,41 @@ const UserProfile: React.FC<UserProfileProps> = ({ fullName, children }) => {
   };
 
   return (
-    <MUIStack direction="row" spacing={1} alignItems="center">
-      <H2
-        variant="h6"
-        sx={{
-          display: { xs: "none", sm: "block" },
-          maxWidth: {
-            sm: "7.5rem",
-            md: "16rem",
-          },
-          overflow: "hidden",
-          whiteSpace: "nowrap",
-          textOverflow: "ellipsis",
-        }}
-        title={fullName}
-      >
-        {fullName}
-      </H2>
-      <MUIAvatar
-        variant="circular-outlined"
-        sx={{ width: 32, height: 32, borderColor: "inherit" }}
-      >
-        <UserIcon color="inherit" />
-      </MUIAvatar>
-      <MUIIconButton
-        aria-label="user-profile"
-        color="inherit"
+    <>
+      <MUIStack
```

Comment:
Can we raise a ticket to change the from a stack to a button to improve accessibility. I have just realised there is an additional onClick event.

Reply:
Done: https://telicent.atlassian.net/browse/TELFE-725.